src/widgets/index/index.ts

```diff
@@ -342,6 +344,105 @@ const index = (props: IndexProps): Index => {
         // it at the index level because it's either: all of them or none of them
         // that are stalled. The queries are performed into a single network request.
         instantSearchInstance.scheduleStalledRender();
+
+        if (__DEV__) {
+          type StateToWidgets = {
+            [TParameter in keyof IndexUiState]: Array<Widget['$$type']>;
```

Comment:
The rule `@typescript-eslint/generic-type-naming` should also allow `^K[A-Z][a-zA-Z]+$` for `KParameter` here.